client/tailscale/localclient.go

```diff
@@ -813,6 +815,30 @@ func (lc *LocalClient) EditPrefs(ctx context.Context, mp *ipn.MaskedPrefs) (*ipn
 	return decodeJSON[*ipn.Prefs](body)
 }
 
+func (lc *LocalClient) GetNetMap(ctx context.Context) (*netmap.NetworkMap, error) {
+	body, err := lc.get200(ctx, "/localapi/v0/netmap")
+	if err != nil {
+		return nil, err
+	}
+	var nm netmap.NetworkMap
+	if err := json.Unmarshal(body, &nm); err != nil {
+		return nil, fmt.Errorf("invalid netmap JSON: %w", err)
+	}
+	return &nm, nil
+}
```

Comment:
You. mentioned that you don't want to watch the IPN bus; given that we have the `NotifyInitialNetMap` flag to ensure that the client can get a netmap, would you mind expanding on the reason that you want to do this as a LocalAPI method vs. doing that?

Reply:
... the reason is that Andrea didn't know about `NotifyInitialNetMap`, fixing :P

Reply:
This also shouldn't have been added to this PR, the proper PR is https://github.com/tailscale/tailscale/pull/13351

Reply:
Fixed.